src/resources/elements/abstract/field.js

```diff
@@ -98,9 +108,18 @@ export class Field {
     this.index = args.index;
     this.parent = args.parent;
     this.showValueInParent = args.showValueInParent;
+    this.hideValueIfEmpty = args.hideValueIfEmpty;
     return this;
   }
 
+  /**
+   * Check if this field is empty. By default, this returns false, but all field
+   * implementations should override this.
+   */
+  isEmpty() {
+    return false;
```

Comment:
If this defaulted to true, it would force field implementations to add their own isEmpty() implementations. I think this would be very useful.